ansible/install-apps.yml

```
@@ -0,0 +1,34 @@
+---
+
+- name: Install OS-level apps
+  hosts: all
+  become: yes
+  vars:
+  - apt_file: /etc/apt/sources.list.d/google-chrome.list
+  - apt_file_Git: 
+  tasks:
+  - name: Does the Google apt file exist?
+    command: test -f {{apt_file}}
```

First of all, it's usually better readable if you space-wrap the contents of jinj2 expression:
```suggestion
    command: test -f {{ apt_file }}
```